coretex/cli/modules/node.py

```diff
@@ -90,14 +99,17 @@ def start(dockerImage: str, config: Dict[str, Any]) -> None:
         if initScript is not None:
             volumes.append((str(initScript), "/script/init.sh"))
 
+        limitedCpus, limitedRam, limitedSwap = docker.dockerInfo()
+        validateResources(config, limitedCpus, limitedRam, limitedSwap)
+
         docker.start(
             config_defaults.DOCKER_CONTAINER_NAME,
             dockerImage,
             config["allowGpu"],
-            config["nodeRam"],
-            config["nodeSwap"],
+            limitedRam if config["nodeRam"] >= limitedRam and limitedRam > 0 else config["limitedRam"],
```

Comment:
User should not be able to start the node at all if the configuration is not valid